docs/guides/burst_insar_product_guide.md

```diff
@@ -186,19 +186,19 @@ and then repackaged into reference and secondary
 [ESA SAFE](https://sentiwiki.copernicus.eu/web/safe-format "SAFE Format" ){target=_blank} 
 files using the 
 [`burst2safe`](https://github.com/ASFHyP3/burst2safe "burst2safe Python package" ){target=_blank} package. 
-Repackaging the burst SLC data into two ESA SAFE files allows the sets of reference and secondary bursts to be 
-processed with ISCE2 as if they were a pair of full IW SLC files from ESA.
+This repackaging allows the sets of reference and secondary bursts to be processed with ISCE2 as if they were a 
+pair of full IW SLC files from ESA.
 
 ##### Considerations for Selecting Input Bursts
 
 A number of conditions need to be met when selecting the sets of bursts to package into the 
-reference and secondary files: 
+reference and secondary SAFE files: 
 
 - Sets of bursts can contain 1-15 bursts
 - There must be the same number of bursts in the secondary set as there are in the reference set
 - All bursts in both the reference and secondary sets must have the same polarization
-  - Only co-polarized inputs are supported
-  - All bursts must be either VV or HH (not VH or HV)
+    - Only co-polarized inputs are supported
+    - All bursts must be either VV or HH (not VH or HV)
 - Pairwise bursts in the reference and secondary sets must have the same burst number, swath number, and polarization
```

Comment:
```suggestion
- Pairwise bursts in the reference and secondary sets must have the same burst number and path number
```

Reply:
Is the swath number the same as the path? Or is this referring to the subswath? 
Also, we hammer the polarization in the previous bullets, so maybe we don't need to include it again here.